exercises/carmen-salas/short-response.md

```diff
@@ -0,0 +1,16 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    *Function parameters go in the parenthesis of function which is a place holder for the arguments. Arguments are values inserted in function parameters when the function is called. 
+
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments? 
+    *When you call a function missing argument the function will return its value with ‘undefined’.  If you call a function with more than one argument it will ignore the other arguments.
```

Comment:
It won't _return_ its value with `undefined`. The missing argument will be passed into the function as `undefined`. Additionally, you made a mistake in sentence two. The function doesn't ignore arguments when a function is passed more than one, only when passed more than it needs. If a function is declared with three parameters, it will only ignore arguments after the third. Make sense?